src/System/Ewald.jl

```diff
@@ -21,24 +21,42 @@ function Ewald(sys::System{N}) where N
     return Ewald(A, μ, ϕ, FA, Fμ, Fϕ, plan, ift_plan)
 end
 
-# Clone all mutable state within Ewald. Note that `A`, `FA` are immutable data.
-# We're also treating FFTW plans as immutable. This is not 100% correct, because
-# plans mutably cache inverse plans, which may possibly lead to data races in a
-# multithreaded context. Unfortunately, FFTW does not yet provide a copy
-# function. For context, see https://github.com/JuliaMath/FFTW.jl/issues/261.
+# Ideally, this would clone all mutable state within Ewald. Note that `A`, `FA`
+# are immutable data. A blocker is that FFTW plans cannot currently be copied,
+# and it is not 100% clear whether they can be treated as immutable. For
+# example, they cache inverse plans, which may possibly lead to data races in a
+# multithreaded context. See discussion at
+# https://github.com/JuliaMath/FFTW.jl/issues/261.
 function clone_ewald(ewald::Ewald)
+    error("Not supported")
```

Comment:
So this function is just a placeholder for the time being?

Reply:
Yes. Maybe we can restore it when https://github.com/JuliaMath/FFTW.jl/issues/261 lands. Leaving it uncommented assists VSCode find/replace in case of a rename.